src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -609,10 +609,14 @@ private static boolean supportsPortFileName(String war) throws IOException {
 
     private static int readPort(File portFile) throws IOException {
         String s = FileUtils.readFileToString(portFile, StandardCharsets.UTF_8);
+
+        // Work around to non-atomic write of port value in Winstone releases prior to 6.1.
+        // Todo When Winstone 6.2 has been widely adopted, this can be deleted.
```

Comment:
generally todo is all caps
```suggestion
        // TODO When Winstone 6.2 has been widely adopted, this can be deleted.
```